### ISSUE TITLE
Can`t install on OpenBSD

### ISSUE THREAD

**Issue by DavidAlphaFox** [yoriyuki/Camomile#73]
I am using opam 2.0.0 with Ocaml 4.07.1 on OpenBSD 6.3.
When I try to install camonile, opam give me the errors below

    > [ERROR] The compilation of camomile failed at "/home/david/.opam/4.07.1/bin/jbuilder build -p camomile -j 1".                                                                
    > 
    > #=== ERROR while compiling camomile.1.0.1 =====================================#
    > # context     2.0.0 | openbsd/x86_64 | ocaml-base-compiler.4.07.1 | https://opam.ocaml.org#da96e33f                                                                          
    > # path        ~/.opam/4.07.1/.opam-switch/build/camomile.1.0.1
    > # command     ~/.opam/4.07.1/bin/jbuilder build -p camomile -j 1
    > # exit-code   1
    > # env-file    ~/.opam/log/camomile-8942-0a5907.env
    > # output-file ~/.opam/log/camomile-8942-0a5907.out
    > ### output ###
    > # [...]
    > # Warning : normalization option is not supported
    > # camomilelocaledef Camomile/locales/tt.mar
    > # Warning : normalization option is not supported
    > # camomilelocaledef Camomile/locales/ur.mar
    > # Warning : normalization option is not supported
    > # camomilelocaledef Camomile/locales/uz.mar
    > # Warning : normalization option is not supported
    > # camomilelocaledef Camomile/locales/vi.mar
    > # Warning : normalization option is not supported
    > # camomilelocaledef Camomile/locales/zh__PINYIN.mar (exit 2)
    > # (cd _build/default/Camomile && tools/camomilelocaledef.exe --file locales/zh__PINYIN.txt locales)                                                                          
    > # Fatal error: exception Stack overflow
    > 
    > 
    > 
    > <><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
    > +- The following actions failed
    > | - build camomile 1.0.1
    > +- 
    > - No changes have been performed
    > 

contents of " ~/.opam/log/camomile-8942-0a5907.out"

    File "Camomile/jbuild", line 4, characters 0-96:
    4 | (rule (with-stdout-to installConfig.ml
    5 |        (echo "let share_dir = \"/usr/share/camomile\"")))
    Warning: File installConfig.ml is both generated by a rule and present in the source tree.
    As a result, the rule is currently ignored, however this will become an error in the future.
    To keep the current behavior and get rid of this warning, add a field (fallback) to the rule.
    ocamlyacc Camomile/internal/uReStrParser.{ml,mli}
    4 shift/reduce conflicts.
    camomilelocaledef Camomile/locales/ar.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/az.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/div.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/el.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/fa.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/gu.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/he.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/hi.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/hi__DIRECT.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/ja.mar
    Warning : strength option is not supported
    camomilelocaledef Camomile/locales/ka.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/kk.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/kn.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/ky.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/mn.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/mr.mar
    camomilelocaledef Camomile/locales/mr.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/pa.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/sa.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/syr.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/ta.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/te.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/th.mar
    Warning : file loading is not supported.
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/tt.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/ur.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/uz.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/vi.mar
    Warning : normalization option is not supported
    camomilelocaledef Camomile/locales/zh__PINYIN.mar (exit 2)
    (cd _build/default/Camomile && tools/camomilelocaledef.exe --file locales/zh__PINYIN.txt locales)
    Fatal error: exception Stack overflow

**Comment by DavidAlphaFox** [comment]
I find out the reason.
On OpenBSD, the `stacksize-cur` in login.conf is 4mb that will cause stackoverflow.
After I modify it to 8mb, `camomile` is compiled and installed.

**Comment by devopsotrator** [comment]
just for seekers -- problem is here and solution still plays well.
OpenBSD 6.5 (-current) GENERIC.MP#846 amd64

**Comment by ffuentese** [comment]
I modified it and even increased it more to  16M and nothing.

**Comment by mndrix** [comment]
I resolved the stack overflow (camomile 1.0.2, OpenBSD 7.0, OCaml 4.13.1) by increasing ulimits:

```
$ ulimit -n "$(ulimit -H -n)" -l "$(ulimit -H -l)" -s "$(ulimit -H -s)"
```

The adjustment to `-n` was probably not necessary.
